### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by UlrichGoebel** [Textualize/textual#4941] [General]
Many thanks to the community and all its helpers!

Even if it's a beginner's question, or if you've just overlooked or simply not found the right place in the documentation, you get very helpful tips from good and patient developers here.

This complements the wonderful textual tool with people who care about it and are obviously happy to help developers like me.
That's a great help, thank you again!

**Comment by willmcgugan** [comment]
Glad you're enjoying it. You might want to join the devs on the Discord server.
